astropy/visualization/tests/test_norm.py

```diff
@@ -238,15 +238,6 @@ def test_imshow_norm():
 
     imshow_norm(image, ax=ax, vmin=0, vmax=1)
 
-    # Note that the following is deprecated in Matplotlib 3.2
```

Comment:
I think we should keep the tests until the keyword is actually removed. Although we should add a `pytest.warns` to ensure the deprecation warning is raised correctly too.

Reply:
Ah yes, thanks!  Done.